scripts/gen_regex.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import unicodedata
+from ftfy import chardata
+import pathlib
+from pkg_resources import resource_filename
+
+
+DATA_PATH = pathlib.Path(resource_filename('wordfreq', 'data'))
+
+
+def _emoji_char_class():
+    """
+    Build a regex for emoji substitution.  We create a regex character set
+    (like "[a-cv-z]") matching characters we consider emoji.
+    """
+    emoji_file = DATA_PATH / 'emoji.txt'
+
+    def accept(c):
+        x = ord(c)
+        return chardata.CHAR_CLASS_STRING[x] == '3' and \
+                x >= 0x2600 and x != 0xfffd
```

Comment:
This can dispense with the call to `ord()` by directly comparing `c` to `\u2600` and `\ufffd`.